src/config/dynamic-theme-fixes.config

```diff
@@ -2863,6 +2863,42 @@ button
 
 ================================
 
+colors.dopely.top
+
+INVERT
+div[class*="navbar__container"]
+div[class="navbar__buttons"]
+div[class="posts-container"]
+div[class*="WhatsNew__Bloks"]
+div[class*="-slider"]
+nav.navbar
+.footer
+.toner-filter-button
+.toner-number-selector-container
+
+CSS
+img[src*=".svg"]:not([alt*="color wheel icon"]):not([class*="dropdown"]):not([src="/static/allInOne.fd51fb9d.svg"]):not([bottom-toolbar-left-item]):not(.ToolsLanding__image),
+div[class*="assetsbar-mobile-block"],
+.contrast-checker-like-icon {
+    filter: invert(50%) saturate(100%) !important;
+}
+div[class$="toner-container"],
+.bottom-toolbar-left-item {
+    background-color: var(--darkreader-neutral-background) !important;
+}
+div[class*="navigation"],
+div[class*="userNavigation"],
+div[class="navbar__buttons"],
+div[class*="controller-container"] {
+    color: var(--darkreader-neutral-background) !important;
+}
+
+IGNORE INLINE STYLE
+span(colors)
```

Comment:
This is a invalid CSS selector.

Reply:
How would I word it in CSS in that case? (For example on the [palette part of the site](https://colors.dopely.top/palette-generator/)) and I'm struggling to prevent the colors from getting inverted and I can't find the anything for the color tiles outside their "ExpandPalette" class name and colors are their background-color.  Because I am stumped

(First image is the inverted result I **don't** want, second image is the result I **do** want)
![color_result](https://user-images.githubusercontent.com/18730519/141801944-067bf918-bf3d-4df5-9d58-0e1fc4431f94.JP
![color_want](https://user-images.githubusercontent.com/18730519/141801981-dfa70d6b-6ba6-4e49-a761-b3963a932f8d.JPG)
G)

Reply:
What about using the `.PaletteGenerator__item--insideItem *` selector? It should cover the case.

Reply:
> .PaletteGenerator__item--insideItem *

I tried that, did nothing in INVERT, it changed something when I put it in IGNORE INLINE and CSS (as:
> .PaletteGenerator__item--insideItem * {
+    filter: invert(0%) !important;
+ }

But all that did was invert the color for the text labels of the color containers

Reply:
Ok so,
> CSS
> .palette-generator-container * {
>     --darkreader-inline-bgcolor: initial !important;
> }
made all of the colors black (and only the colors), but I just need to be able to reword the statement so it reverts back to the un-inverted color it originally had

Reply:
Hmm, what about using `.ExpandPalette *` in the IGNORE INLINE STYLE?

Reply:
didn't anything either

Reply:
Did you refresh the site after applying?

Reply:
> Did you refresh the site after applying?

Ahh yeah it worked, I also was implying .ExpandPalette would still be in CSS, but changed it to the .palettegen
lemme fix up the other generators with the same technique and clean up the code. Thanks! (just i reply the fix once done?)

Reply:
> (just i reply the fix once done?)

I automatically get a notification if a edit is made.